clients/cobol-lsp-vscode-extension/src/__tests__/services/SettingsTest.spec.ts

```diff
@@ -25,6 +25,7 @@ beforeAll(() => {
 
 // TODO: this is horrifying as well
 jest.mock("vscode", () => {
+  // eslint-disable-next-line @typescript-eslint/no-require-imports
   const Uri = require("../../__mocks__/UriMock").Uri;
```

Comment:
import?

Reply:
same as in the `CopybookResolveURITest.spec.ts` - the tests fail if the Uri is imported instead of required